docs/source/building-supergraphs/router.mdx

```diff
@@ -80,29 +86,31 @@ On startup, the gateway processes your `supergraphSdl`, which includes routing i
 In the above example, we provide a _static_ supergraph schema to the gateway. This approach requires the gateway to restart in order to update the supergraph schema. This is undesirable for many applications, so we also provide the ability to update the supergraph schema dynamically.
 
 ```js title="index.js"
-const { ApolloServer } = require('apollo-server');
+const { ApolloServer } = require('@apollo/server');
+const { startStandaloneServer } = require('@apollo/server/standalone');
 const { ApolloGateway } = require('@apollo/gateway');
 const { readFile } = require('fs/promises');
 
-let supergraphUpdate;
-const gateway = new ApolloGateway({
-  async supergraphSdl({ update }) {
-    // `update` is a function that we'll save for later use
-    supergraphUpdate = update;
-    return {
-      supergraphSdl: await readFile('./supergraph.graphql', 'utf-8'),
-    }
-  },
-});
+async function startApolloGateway() {
```

Comment:
Same here, should call this function in the snippet so it runs

Reply:
There are a number of these which I didn't leave comments on

Reply:
(I'll fix these with whatever we decide above)